src/dotnet/ResourceManagerResearch/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Azure.Identity;
```

Comment:
Is the research still required? (in the main version). Perhaps we can leave this somewhere else?

Reply:
deleted - also the integration test, because it is not doing anything